### DISCUSSION TITLE
How to modify UI when I visit backend server

### DISCUSSION THREAD

**Discussion by da-da-da** [nocodb/nocodb#6849] [Q&A]
Hi, I'm new in frontend development. Firstly, I follow the Development Setup🥸

I have modify UI like html or css, then I use `pnpm start:frontend`, and visit localhost:3000. I can see the change.

But when I start the backend server（`pnpm start:backend`）， there are nothing changed in localhost:8080.

**Comment by o1lab** [comment]
Whilst in development - please use :3000 address. That should include your backend change too.

**Reply by da-da-da** [reply]
Thank you. Why is the modification invalid when accessing 8080? 
If I need to build local image in docker or release, what can I do to keep my modification of frontend.

**Reply by da-da-da** [reply]
plus, I have use `build-local-docker-image.sh` to build local image and run the project in cantainer, but the modification doesn't work.
